content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,178 @@
+
+### 📱 Mobile first
+
+Touch support and mobile-optimized layout [were](https://github.com/excalidraw/excalidraw/pull/787) [first](https://github.com/excalidraw/excalidraw/pull/788) [added](https://github.com/excalidraw/excalidraw/pull/790) by [j-f1](https://github.com/j-f1). This includes the creation of a toolbar for mobile devices that displays relevant controls while still leaving most of the screen free for the canvas.
+
+### 📚 Library
+
+The library was [first implemented](https://github.com/excalidraw/excalidraw/pull/1787) by [petehunt](https://github.com/petehunt). Later, after adding support for exporting the library to a file, we eventually introduced a public directory for libraries. Visit [libraries.excalidraw.com](https://libraries.excalidraw.com) for more.
+
+https://twitter.com/dbs_sticky/status/1340349749086580736
+
+### 📊 Excalicharts
+
+While we aim to keep Excalidraw simple to use, sometimes we hide little easter eggs that you need to find out for yourself (or find some hints by following our [Twitter account](https://twitter.com/excalidraw)). For example, you can copy any two-column dataset from a spreadsheet, or comma separated values (CSV) from a text file, and paste them into Excalidraw to quickly produce a chart. The [first implementation](https://github.com/excalidraw/excalidraw/pull/1723) was done by [petehunt](https://github.com/petehunt) and several [improvements](https://github.com/excalidraw/excalidraw/pull/2495) were made by [lipis](https://github.com/lipis).
+
+### ✍️ Free Draw mode
+
+The [free hand draw](https://github.com/excalidraw/excalidraw/pull/1570) was one of the most [requested](https://github.com/excalidraw/excalidraw/issues/25) features. Thanks to [kbariotis](https://github.com/kbariotis) for implementing this.
+
+https://twitter.com/excalidraw/status/1260287781596794880
+
+### #️⃣ Grid and Stats
+
+From early on people were asking for more precision in their hand drawn diagrams. We complied by adding the [grid support](https://github.com/excalidraw/excalidraw/pull/1788) implemented by [dai-chi](https://github.com/dai-shi) and—inspired by _YouTube's stats for nerds_—our own version of [stats](https://github.com/excalidraw/excalidraw/pull/2453) implemented by [lipis](https://github.com/lipis). Both features could be found under the context menu from the canvas.
```

Comment:
```suggestion
From early on, people were asking for more precision in their hand drawn diagrams. We complied by adding the [grid support](https://github.com/excalidraw/excalidraw/pull/1788) implemented by [dai-shi](https://github.com/dai-shi), and—inspired by _YouTube's stats for nerds_—our own version of [stats](https://github.com/excalidraw/excalidraw/pull/2453) implemented by [lipis](https://github.com/lipis). Both features could be found under the context menu by right-clicking on the canvas.
```

Reply:
```suggestion
From early on people were asking for more precision in their hand-drawn diagrams. We complied by adding the [grid support](https://github.com/excalidraw/excalidraw/pull/1788) implemented by [dai-chi](https://github.com/dai-shi) and—inspired by _YouTube's stats for nerds_—our own version of [stats](https://github.com/excalidraw/excalidraw/pull/2453) implemented by [lipis](https://github.com/lipis). Both features could be found under the context menu from the canvas.

https://twitter.com/excalidraw/status/1275451062414045190
https://twitter.com/excalidraw/status/1335988128608301063
```
Since these features are hidden behind the context menu we might want to embed twitter links